src/clients/starknet-tx/abi/space.json

```diff
@@ -7,74 +7,84 @@
         "type": "felt"
```

Comment:
not really used, but updated it so it functions as a reference to ABI that this code actually expects space to have.